### PR TITLE
Update the EBL directory

### PR DESCRIPTION
**Pull request by bkhelifi**
This PR updates the references to the models, as well as the figure

### CODE REVIEW

**Review by maxnoe — CHANGES_REQUESTED** [review]
There is a typo in the legend for Francesch__i__ni 2017

**Review by maxnoe — APPROVED** [review]

**Comment by bkhelifi** [comment]
Thanks @maxnoe. I am still waiting for the feedback from Finke, Dominguez ans Saldana-Lopez about the licence question.
I propose that we merge this PR once we have all the authors' information.

**Comment by bkhelifi** [comment]
Add the licences are added. This PR is ready to be reviewed... @registerrier

**Review by registerrier — APPROVED** [review]
Thanks @bkhelifi . Looks good to me.

File: ebl/README.md [hunk 1]
```diff
@@ -1,9 +1,31 @@
 # Models for extragalactic background light (EBL)
 
 Available models are:
- - Franceschini 2008 (http://adsabs.harvard.edu/abs/2008A%26A...487..837F)
- - Dominguez 2011 (http://adsabs.harvard.edu/cgi-bin/bib_query?arXiv:1007.1459)
- - Finke 2010 (http://adsabs.harvard.edu/abs/2009arXiv0905.1115F)
+ - Franceschini 2008:
+   - Reference: [Franceschini, A.; Rodighiero, G.; Vaccari, M., 2008A&A...487..837F](http://adsabs.harvard.edu/abs/2008A%26A...487..837F)
+   - Licence: Extragalactic optical-infrared background radiation, its time 
+   evolution and the cosmic photon-photon opacity © 2008 by Alberto Franceschini -
```
Comment (registerrier) [inline ebl/README.md:7]: A question. Is it just the first author here?

File: ebl/README.md [hunk 2]
```diff
@@ -1,9 +1,31 @@
...
+   Stars and Dust" © 2024 by  Justin D. Finke, Soebur Razzaque, Charles D. Dermer is 
+   licensed under [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/).
+ - Dominguez 2011:
+   - Reference:  [Dominguez, A., et al., 2011MNRAS.410.2556D](https://ui.adsabs.harvard.edu/abs/2011MNRAS.410.2556D/abstract)
+   - Licence: Extragalactic background light inferred from AEGIS galaxy-SED-type fractions 
+   © 2011 by Alberto Domínguez et al. is licensed under [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/). 
+ - Franceschini 2017
+   - Reference: [Franceschini, A., Rodighiero, G. 2017A&A...603A..34F](https://ui.adsabs.harvard.edu/abs/2017A%26A...603A..34F)
+   - Licence: Extragalactic optical-infrared background radiation, its time 
+   evolution and the cosmic photon-photon opacity © 2017 by Alberto Franceschini -
```
Comment (registerrier) [inline ebl/README.md:21]: Same.

**Review by registerrier — APPROVED** [review]
Thanks @bkhelifi !

**Pull request merged**